application/helpers/expressions/em_cache_helper.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/**
+ * Discussion here: https://bugs.limesurvey.org/view.php?id=14859
+ *
+ * @since 2019-05-23
+ * @author Olle Haerstedt
+ */
+class EmCacheHelper
```

Comment:
Currently : usage os only to set surveyinfo ? But have idea to extend it after ?

Reply:
The helper class? Sure, lots of other functionality. See later commits.